### ISSUE TITLE
Problem with version 1.2.6 and Webpack 2.1.0-beta.20

### ISSUE THREAD

**Issue by code-chimp** [reactabular/reactabular#189]
I am getting strange errors trying to use reactabular 1.2.6 (via npm) with webpack 2.1.0-beta.20. I was able to use1.2.1 previously and it seemed to work fine, but the latest is throwing a bunch of the following errors:

```
ERROR in ../~/reactabular/~/reactabular-sticky/src/index.jsx
Module parse failed: C:\work\lab\react\reactabular-test\node_modules\reactabular\node_modules\reactabular-sticky\src\index.jsx Unexpected token (12:30)
You may need an appropriate loader to handle this file type.
|   }
|   render() {
|     const { style, tableBody, ...props } = this.props;
|
|     return (
 @ ../~/reactabular/src/index.js 2:0-45
 @ ./js/components/App.js
 @ ./js/index.js
```

pretty much like that for all of the sub-packages like reactabular-select, reactabular-search-field, etc. Anything off the top of your head that I might have missed? I was just trying to use the basic table first

```javascript
import { Table } from 'reactabular'
```

so nothing really crazy.  I have slapped together [a quick demo project](https://github.com/code-chimp/reactabular-test) so hopefully you can see the errors I am getting and maybe see where I went wrong.  Thanks!

**Comment by bebraw** [comment]
I think I see a solution now. I need to apply my Babel plugins and generate pure ES6 distribution version as that's what the other tooling, webpack 2 included, expects.

**Comment by bebraw** [comment]
This will be fixed 2.0. Thanks for the report!
